CODEOWNERS

```diff
@@ -2,7 +2,47 @@
 /erroranalysis @gaugup @imatiach-msft
 
 # Responsible AI widgets package
-/raiwidgets @imatiach-msft @xuke444 @zhb000 @vinuthakaranth @RubyZ10 @tongyu-microsoft @romanlutz
+/raiwidgets @imatiach-msft @xuke444 @romanlutz
 
 # Responsible AI SDK package
-/responsibleai @gaugup @imatiach-msft @riedgar-ms @xuke444 @kicha0
\ No newline at end of file
+/responsibleai @gaugup @imatiach-msft @riedgar-ms @xuke444 @kicha0
+
+# Responsible AI SDK package
+/responsibleai @gaugup @imatiach-msft @riedgar-ms @xuke444 @kicha0
+
+# workspace settings
+#/* @xuke444
```

Comment:
Shall we also add code owners for raiutils?

Reply:
I didn't add any there since it is small and I am not clear what it will end up with later yet.

Currently only our group can approve pr, so even without adding any approver there, only 10 of us can review and approve.